### ISSUE TITLE
Support SVG -> PDF

### ISSUE THREAD

**Issue by domoritz** [vega/vega-render-service#7]

**Comment by hungngo97** [comment]
By this, do you mean capturing the current graph in interactive mode on the editor and get its SVG? Then parse the SVG to PDF?

**Comment by domoritz** [comment]
Yes. The challenge is still that the server won't have fonts. Some charts may look slightly different. Also, spacing is calculated by Vega based on the fonts in the browser so there may be problems because of that.

**Comment by hungngo97** [comment]
But can we take a snapshot of the SVG in the middle of a graph interaction?

**Comment by domoritz** [comment]
Yes, at any time. You can get it from the `view` object.
